themes/default/content/product/_index.md

```diff
@@ -12,80 +12,115 @@ subheading: |
-                    by leveraging the features of each language.
+                    Declarative IaC in TypeScript/JavaScript, Python, Go, C#, Java, and YAML. Deep runtime integrations and no proprietary frameworks.
 
-              - title: Build on any cloud
+              - title:
                 icon: global
                 icon_color: yellow
                 description: |
-                    Access the full breadth of services in AWS, Azure, GCP, and [60+ providers](/registry/) through
-                    a complete and consistent SDK interface.
+                    Use your favorite IDEs with autocomplete, in-line documentation, and a natural development experience.
 
-              - title: Create reusable infrastructure
+              - title:
                 icon: puzzle
                 icon_color: yellow
                 description:
-                    Build and share cloud infrastructure with reusable [Pulumi Packages](/product/packages/), which
-                    encapsulate your cloud architectures and best practices.
+                    Run rapid unit tests with any supported framework to validate your infrastructure code before deployment.
 
-        - description: Deploy cloud infrastructure and applications together
+        - description: Consume infrastructure as code in any language
           features:
-              - title: Multiple deployment options
+              - title:
                 icon: rocketship
                 icon_color: salmon
                 description: |
-                    Deploy infrastructure interactively with a CLI, programmatically with Pulumi’s
-                    [Automation API](/automation/), or through your [CI/CD process](/docs/guides/continuous-delivery/).
+                    Build faster by reusing Pulumi libraries and CDK constructs, and create your own libraries as multi-language Pulumi Packages.
 
-              - title: Deployments as Code
+              - title:
                 icon: gear
                 icon_color: salmon
                 description: |
-                    Run deployments from your application code at runtime with [Automation API](/automation/). Create infrastructure APIs, custom platforms, and CLIs.
+                    Develop modern cloud applications faster while using well-architected best practices with the Crosswalk for AWS library.
 
-              - title: Preview and test changes
+              - title:
                 icon: eye
                 icon_color: salmon
                 description: |
-                    Test and validate infrastructure with standard [unit test frameworks](/docs/guides/testing/#unit-testing) and
-                    [integration tests](/docs/guides/testing/integration/). Preview changes before deploying.
+                    Convert your existing infrastructure code to Pulumi (TF, ARM, CFN, K8s, CRD, Helm) and import existing resources into Pulumi code.
```

Comment:
link "convert" to https://www.pulumi.com/docs/converters/

and "import" to https://www.pulumi.com/docs/guides/adopting/import/